spec/unit/helpers/check_helper_spec.rb

```diff
@@ -50,6 +50,42 @@
     expect { params[:test_param_three] }.to raise_error(/not defined in the specification/)
   end
 
+  context 'when created from stringified hash' do
+    let(:params) do
+      raw_params = { 'update_preference' => { 'keep_data' => false,
+                                              'allow_cascade_drops' => true } }
+      GoodData::LCM2.convert_to_smart_hash(raw_params)
+    end
+
+    let(:other) do
```

Comment:
Similar code found in check_helper_spec.rb:54, check_helper_spec.rb:60 (mass = 32)